docs/configuration-properties.md

```diff
@@ -81,7 +81,61 @@ See https://github.com/Netflix/SimianArmy/wiki/Chaos-Settings for a detailed des
 
 ## Janitor Monkey Properties
 
-Janitor Monkey is disabled and cannot be configured at the moment.
+| Key | Default |
+| --- | ------- |
+| simianarmy/janitor/enabled | true
```

Comment:
This is not "true".

Also, could you please add a trailing "|" in every row for consistency?